shape-the-future/aspire.md

```diff
@@ -0,0 +1,11 @@
+---
+title: NServiceBus and Aspire
+summary: How to use NServiceBus with Aspire
+reviewed: 2024-09-16
+---
+
+[.NET Aspire](https://learn.microsoft.com/en-us/dotnet/aspire/get-started/aspire-overview) is an opinionated, cloud-ready stack for building observable, production ready, distributed applications.​
+
+NServiceBus endpoints can run under an Aspire AppHost orchestrator, we have [a sample that shows you how it all works](/samples/hosting/aspire/).
+
+If you are using NServiceBus with .NET Aspire today, [tell us what's missing](https://github.com/Particular/NServiceBus/issues/6941).
```

Comment:
We should update the issue to include the leading questions:
- Do you use https://learn.microsoft.com/en-us/dotnet/aspire/deployment/manifest-format
- Are you currently using Aspire, if so is it just for development or also for production deployments
- What other Aspire integrations are you using
- Do you use health checks - https://learn.microsoft.com/en-us/dotnet/aspire/fundamentals/health-checks
- Do you use telemetry